docs/0133.md

```diff
@@ -0,0 +1,381 @@
+
+### Details
+
+**Rule**: aep-133-required-params
+
+This rule looks at the parameters of "post" operations on a path that does not have a ":" in its final segment
+(indicating a custom operation), and complains if it finds any required parameters that are not path parameters (which
+must be required).
+
+### Examples
+
+**Incorrect** code for this rule:
+
+```yaml
+paths:
+  '/test1':
+    post:
+      parameters:
+        - name: force
+          in: query
+          required: true
+          schema:
+            type: boolean
+```
+
+**Correct** code for this rule:
+
+```yaml
+paths:
+  '/test1/{id}/test2':
+    post:
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: force
+          in: query
+          schema:
+            type: boolean
+```
+
+### Disabling
+
+If you need to violate this rule for a specific operation, add an "override" to the Spectral rule file for the specific
+file and fragment.
+
+```yaml
+overrides:
+  - files:
+      - 'openapi.json#/paths/test1/post/parameters/0'
+    rules:
+      aep-133-required-params: 'off'
+```
+
+If you need to violate this rule for an entire file, add an "override" to the Spectral rule file for the specific file
+without a fragment.
+
+## Create methods should have no unknown optional parameters
+
+This rule checks that Create methods have no optional parameters other than parameters defined in [AEP-133] or other
+AEPs.
+
+### Details
+
+**Rule**: aep-133-unknown-optional-params
+
+This rule looks at the parameters of "post" operations on a path that does not have a ":" in its final segment
+(indicating a custom operation), and complains if it finds any optional query parameters that are described in the AEPs.
+Currently the only allowed optional query parameter is `id`, described in [AEP-133].
+
+### Examples
+
+**Incorrect** code for this rule:
+
+```yaml
+paths:
+  '/test1':
+    post:
+      parameters:
+        - name: force
+          in: query
+          schema:
+            type: boolean
+```
+
+**Correct** code for this rule:
+
+```yaml
+paths:
+  '/test1/{testId}/test2':
+    post:
+      parameters:
+        - name: testId
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: id
+          in: query
+          schema:
+            type: string
+```
+
+### Disabling
+
+If you need to violate this rule for a specific operation, add an "override" to the Spectral rule file for the specific
+file and fragment.
+
+```yaml
+overrides:
+  - files:
+      - 'openapi.json#/paths/test1/post/parameters/0'
+    rules:
+      aep-133-unknown-optional-params: 'off'
+```
+
+If you need to violate this rule for an entire file, add an "override" to the Spectral rule file for the specific file
+without a fragment.
+
+## A standard Create method must accept the resource in the request body
```

Comment:
nit: should these headers be the name of the parameter, e.g. `aep-133-request-body`? Then it will be easier to find in the docs, but including it as a an anchor link (like #aep-133-request-body).

Reply:
I think that would work! I'd personally name the header `aep-133-request-body`, but your approach works great! Feel free to resolve my comment after it's added.

Reply:
I wanted to try your idea so I made those changes and pushed a new commit. This would be a change to the way the docs are currently structured (for the rules that have already been merged), but I kinda like this structure so if you agree we'll go with this and I'll update the other doc files.

Reply:
Looks great! thanks for changing that. Agreed it's fine to follow up with updates to the other rules in a follow-up PR.